api.planx.uk/modules/file/controller.ts

```diff
@@ -140,8 +152,8 @@ export const publicDeleteController: DeleteController = async (
   const filePath = buildFilePath(fileKey, fileName);
 
   try {
-    const { isPrivate } = await getFileFromS3(filePath);
-    if (isPrivate) throw Error("Bad request");
+    const file = await getFileFromS3(filePath);
+    if (!file || file.isPrivate) throw Error("Bad request");
```

Comment:
is it useful to differentiate between cases here? e.g. if someone is trying to delete a public file, but it in fact does not exist because it was never uploaded in the first place, it might be misleading to throw and error which says `Failed to delete...`?

Reply:
Good spot, fixed in [c926c52](https://github.com/theopensystemslab/planx-new/pull/4542/commits/c926c5206ccd7bf8983ba876e1300dcf6f0d7119)